### ISSUE TITLE
there may have some erros in crf 

### ISSUE THREAD

**Issue by jabgle** [warmspringwinds/tensorflow_notes#11]
softmax = final_probabilities.squeeze()
softmax = processed_probabilities.transpose((2, 0, 1))

i think it should  be：

 processed_probabilities= final_probabilities.squeeze()
softmax = processed_probabilities.transpose((2, 0, 1))
 am i right?
